Topic3TDD/src/exc1/NumberConverter.java

```diff
@@ -0,0 +1,87 @@
+package exc1;
+
+public class NumberConverter {
+
+	private static final String[] units = { "", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten", "eleven",
+			"twelve", "thirteen", "fourteen", "fifteen", "sixteen", "seventeen", "eighteen", "nineteen" };
+
+	private static final String[] tens = { "", // 0
+			"", // 1
+			"twenty", // 2
+			"thirty", // 3
+			"forty", // 4
+			"fifty", // 5
+			"sixty", // 6
+			"seventy", // 7
+			"eighty", // 8
+			"ninety" // 9
+	};
+
+	private String convert(int n) {
+
+		if (n < 0) {

```

Comment:
According with the point one this condition isn't necessary, there is no negative dollars